bundler/spec/support/builders.rb

```diff
@@ -245,7 +245,8 @@ def check_test_gems!
     end
 
     def update_repo(path, build_compact_index: true)
-      if path == gem_repo1 && caller.first.split(" ").last == "`build_repo`"
+      exempted_caller = RUBY_VERSION.start_with?("3.4") ? "#{Module.nesting.first}#build_repo" : "build_repo"
```

Comment:
@nobu Is this line safe like Ruby 3.5? Should we use `RUBY_VERSION >= '3.4.0.dev'` instead of this?

Reply:
Good point, it seems like a change towards more informative backtraces, so I'd assume it's there to stay. I think it's best to use `RUBY_VERSION >= '3.4.0.dev'` 👍.